python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
-   :math:`\uX`. The elements are the followings :
-   :math:`(\Cov \uX)_{ij} = \Expect{\left(X^i - \Expect{X^i} \right)\times\left(X^j - \Expect{X^j} \right)}`
+In the remainder, we denote by :math:`\Cov \uX`  the covariance matrix of :math:`\uX`, defined by:
 
--  :math:`\underline{\nabla} h(\muX) = \: ^t \left( \frac{\partial y}{\partial x^j}\right)_{\ux\: =\: \muX} = \: ^t \left( \frac{\partial h(\ux)}{\partial x^j}\right)_{\ux\: =\: \muX}`
-   is the gradient vector taken at the value :math:`\ux = \muX` and
-   :math:`j=1,\ldots,n_X`.
+.. math::
 
--  :math:`\underline{\underline{\nabla}}^2 h(\ux,\ux)` is a matrix. It
-   is composed by the second order derivative of the output variable
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` taken around :math:`\ux = \muX`. It yields
-   to:
-   :math:`\left( \nabla^2 h(\muX,\muX) \right)_{ij} =\left( \frac{\partial^2 h(\ux,\ux)}{\partial x^i \partial x^j}\right)_{\ux\: =\: \muX}`
+    \Cov \uX = (c_{ij})_{ij}
 
--  :math:`<,>` is a scalar product between two vectors.
 
-| **Approximation at the order 1 - Case** :math:`n_Y=1`
+where:
 
-  .. math::
+.. math::
 
-      \Expect{Y} = h(\muX)
+    c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}
 
 
-  .. math::
+with :math:`c_{ii} = \sigma_i^2`.
 
-      \Var{Y} = \sum_{i,j=1}^{n_X} \frac{\partial h(\muX)}{\partial X^i} . \frac{\partial h(\muX)}{\partial X^j}.(\Cov \uX)_{ij}
+**Case 1:** :math:`Y \in \Rset` , :math:`q=1`:
 
-| **Approximation at the order 2 - Case** :math:`n_Y=1`
+The second order Taylor expansion of :math:`h` around :math:`\ux = \vect{\mu}` is written as:
 
-  .. math::
+.. math::
 
-     \begin{aligned}
-         \begin{split}
-           \Expect{Y} = h(\muX) + \frac{1}{2}. \sum_{i,j=1}^{n_X} \frac{\partial^2 h(\muX,\muX)}{\partial x^i \partial x^j} . (\Cov \uX)_{ij}
-         \end{split}
-       \end{aligned}
+    Y = h(\vect{\mu}) + \sum_{i = 1}^d \left( \frac{\partial h}{\partial x_i }\right)(\vect{\mu})(X_i-\mu_i)
+    + \frac{1}{2} \sum_{i,j = 1}^d \left(\frac{\partial^2 h}{\partial x_i \partial x_j}\right)(\vect{\mu})
+    (X_i-\mu_i)(X_j-\mu_j) + o(||\uX||^2)
 
-| The decomposition of the variance at the order 2 is not implemented.
-  It requires both the knowledge of higher order derivatives of the model
-  and the knowledge of moments of order strictly greater than 2 of the pdf.
 
-| **Case** :math:`n_Y>1`
+If we use the first order expansion, we get:
 
+  .. math::
 
-| The perturbation approach can be developed at different orders
-  from the Taylor decomposition of the random vector :math:`\uY`. As
-  :math:`\uY=h(\uX)`, the Taylor decomposition around :math:`\ux = \muX`
-  at the second order yields to:
+       \begin{array}{lcl}
+            \Expect{Y} & \simeq & h(\vect{\mu})\\
+            \Var{Y} & \simeq & \sum_{i=1}^{d} \sigma_i^2 \left[\left(\frac{\partial h}{\partial x_i}
+            \right)(\vect{\mu}) \right]^2
+       \end{array}
```

Comment:
You wrongly use the `\simeq` symbol which means asymptotically equal (see e.g; this message on [Stack](https://math.stackexchange.com/questions/864606/difference-between-%E2%89%88-%E2%89%83-and-%E2%89%85)) instead of the `\approx` symbol.
```suggestion
            \Expect{Y} & \simeq & h(\vect{\mu})\\
            \Var{Y} & \simeq & \sum_{i=1}^{d} \sigma_i^2 \left[\left(\frac{\partial h}{\partial x_i}
            \right)(\vect{\mu}) \right]^2
```

Reply:
ok